webpack/assets/javascripts/react_app/components/Layout/LayoutReducer.js

```diff
@@ -7,11 +7,13 @@ import {
   LAYOUT_UPDATE_ITEMS,
   LAYOUT_CHANGE_ORG,
   LAYOUT_CHANGE_LOCATION,
+  LAYOUT_LOGOUT,
 } from './LayoutConstants';
 
 const initialState = Immutable({
   items: [],
   isLoading: false,
+  isLoggedIn: true,
```

Comment:
Why do we want to keep track of the login status in the client side? if a user isn't logged in, any request to the server should redirect them to the login page (in the ui) or reply with 401 unauthorized (in the api).
If we start relying on the redux state to determine if the user is logged in or not (and from that determine if certain actions can be done) a malicious user can modify the state and possibly gain access they shouldn't have.

Reply:
I thought it might be useful when the login page switches to react, but its not necessary as of now

Reply:
I would still want the server to make the decision on whether the login page should be displayed and not the data the user can manipulate.